airflow/api_connexion/endpoints/health_endpoint.py

```diff
@@ -14,13 +14,34 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-# TODO(mik-laj): We have to implement it.
-#     Do you want to help? Please look at: https://github.com/apache/airflow/issues/8144
+from airflow.api_connexion.schemas.health_schema import health_schema
+from airflow.configuration import conf
+from airflow.jobs.scheduler_job import SchedulerJob
+from airflow.utils.session import provide_session
 
 
 def get_health():
     """
-    Checks if the API works
+    Return the health of the airflow scheduler and metadatabase
     """
-    return "OK"
+    payload = {
+        'metadatabase': {'status': 'unhealthy'}
+    }
+
+    latest_scheduler_heartbeat = None
+    scheduler_status = 'unhealthy'
+    payload['metadatabase'] = {'status': 'healthy'}
+    try:
+        scheduler_job = SchedulerJob.most_recent_job()
+
+        if scheduler_job:
+            latest_scheduler_heartbeat = scheduler_job.latest_heartbeat.isoformat()
+            if scheduler_job.is_alive():
+                scheduler_status = 'healthy'
+    except Exception as err:
+        payload['metadatabase']['status'] = 'unhealthy'
+
+    payload['scheduler'] = {'status': scheduler_status,
+                            'latest_scheduler_heartbeat': latest_scheduler_heartbeat}
+
```

Comment:
This code is the exact copy of [`airflow.www.views.Airflow.health`](https://github.com/apache/airflow/blob/7f70f6f7207874849ee2d380159ef01bbe950da0/airflow/www/views.py#L192).